### PR TITLE
docs(tutorial): instances snapshots with Serverless Jobs MTA-4622

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: tutorials/snapshot-instances-jobs/index.mdx [hunk 1]
```diff
@@ -0,0 +1,260 @@
+---
+meta:
+  title: Create snapshots of an instance with Serverless Jobs
+  description: This step-by-step tutorial will help you automate the creation of snapshots of your instance using Serverless Jobs 
+content:
+  h1: Create snapshots of an instance with Serverless Jobs
+  paragraph: This step-by-step tutorial will help you automate the creation of snapshots of your instance using Serverless Jobs
```
Comment (ldecarvalho-doc) [inline tutorials/snapshot-instances-jobs/index.mdx:7]:
```suggestion
  title: Create snapshots of an Instance with Serverless Jobs
  description: This step-by-step tutorial will help you automate the creation of snapshots of your Instance using Serverless Jobs 
content:
  h1: Create snapshots of an Instance with Serverless Jobs
  paragraph: This step-by-step tutorial will help you automate the creation of snapshots of your Instance using Serverless Jobs
```

File: tutorials/snapshot-instances-jobs/index.mdx [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- Created a [Container Registry namespace](https://www.scaleway.com/en/docs/containers/container-registry/how-to/create-namespace/).
+- Created an [Instance](/compute/instances/how-to/create-an-instance/)
+- Installed and started the [Docker daemon](https://docs.docker.com/engine/install/) to build the image.
+
+## Creating the snapshot generator
+
+1. run the following command in a terminal to create a `main.go` file that contains the code to create snapshots:
```
Comment (ldecarvalho-doc) [inline tutorials/snapshot-instances-jobs/index.mdx:31]:
```suggestion
1. Run the following command in a terminal to create a `main.go` file that contains the code to create snapshots:
```

File: tutorials/snapshot-instances-jobs/index.mdx [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+      for i, b := range bytes {
+        bytes[i] = letters[b%byte(len(letters))]
+      }
+
+      return string(bytes)
+    }
+    EOF
+    ```
+
+## Building and pushing the imager to Container Registry
```
Comment (ldecarvalho-doc) [inline tutorials/snapshot-instances-jobs/index.mdx:170]:
```suggestion
## Building and pushing the image to Container Registry
```

File: tutorials/snapshot-instances-jobs/index.mdx [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+
+2. Click **Create job**. The job creation wizard displays.
+
+3. Select the **Scaleway** Container Registry, then select your Container Registry namespace from the drop-down list, and the container image and tag.
+
+4. Enter a name or use the automatically generated one.
+
+5. Select the region in which your job will be created.
+
+6. keep the default **resources** values, as this job requires little compute capabilities.
```
Comment (ldecarvalho-doc) [inline tutorials/snapshot-instances-jobs/index.mdx:229]:
```suggestion
6. Keep the default **resources** values, as this job requires little compute capabilities.
```

File: tutorials/snapshot-instances-jobs/index.mdx [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+
+7. Set the **cron schedule** to `* 2 * * *` and select the relevant time zone to run the job every day at 2:00 a.m. Refer to the [cron schedules documentation](/serverless/jobs/reference-content/cron-schedules/) for more information.
+
+8. Define the following environment variables:
+    - `INSTANCE_ID`: the ID of the instance you want to snapshot
+    - `INSTANCE_ZONE`: the [Availabilitiy Zone](/compute/instances/concepts/#availability-zone) of your instance (e.ge. `fr-par-2`)
```
Comment (ldecarvalho-doc) [inline tutorials/snapshot-instances-jobs/index.mdx:235]:
```suggestion
    - `INSTANCE_ZONE`: the [Availabilitiy Zone](/compute/instances/concepts/#availability-zone) of your Instance (e.ge. `fr-par-2`)
```

**Review by ldecarvalho-doc — APPROVED** [review]

**Comment by SamyOubouaziz** [comment]
Modify copy-paste instructions according to Thomas' feedback

File: tutorials/snapshot-instances-jobs/index.mdx (hunk 5 above)
Comment (nerda-codes) [inline tutorials/snapshot-instances-jobs/index.mdx:235]:
```suggestion
    - `INSTANCE_ZONE`: the [Availabilitiy Zone](/compute/instances/concepts/#availability-zone) of your Instance (e.g. `fr-par-2`)
```

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**